### PR TITLE
PENDING: Login Screen

### PR DESCRIPTION
**Pull request by mshago**
Screenshot_1648246786
We need to make some changes to the Container so it can accept custom styles

### CODE REVIEW

File: src/screens/login/LoginScreen.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const LoginScreen = () => {
+  return (
+  <Container background={color.midGray} containerStyle={styles.container}>
+    <TextInput 
+        placeholder={'Email'}
+        style={styles.textInput}/>
+    <TextInput 
+        isSensitive={true}
+        placeholder={'Email'}
```
Comment (lasharela) [inline src/screens/login/LoginScreen.js:14]: password? a placeholder

**Review by lasharela — APPROVED** [review]

File: src/screens/login/LoginScreen.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import {Button, Container, TextInput} from '../../components'
+import { color } from '../../theme'
+import {styles } from './loginScreen.styles'
+
+export const LoginScreen = () => {
+  return (
+  <Container background={color.midGray} containerStyle={styles.container}>
```
Comment (lasharela) [inline src/screens/login/LoginScreen.js:8]: do we need it?

**Review by raynesmike — APPROVED** [review]
I Approve this addition

File: src/screens/login/LoginScreen.js (hunk 2 above)
Comment (lasharela) [inline src/screens/login/LoginScreen.js:8]: remove `background` prop from Contaqiner component

File: src/screens/login/LoginScreen.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <TextInput
```
Comment (lasharela) [inline src/screens/login/LoginScreen.js:9]: <View>
 <TextInput 
        placeholder={'Email'}
        style={styles.textInput}/>
<Text>error in advance</Text>
</View>

**Review by lasharela — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]
The placeholder color is too bright, can't see it. Add `placeholderTextColor={color.darkGray}` to the` <textInput />`. Also don't forget to change the placeholder of the bottom text input to `Password`. Otherwise, it works fine for me.

**Pull request merged**